EIPS/eip-rename_gas_to_mana.md

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
```suggestion
---
eip: 6789
```
Assigning EIP number 6789 as it is the PR number.

Reply:
Resolved via https://github.com/ethereum/EIPs/pull/6789/commits/3433122eba75c4f160773f3439cdf216f191babd